From 22e8b70867d881123465ee6ac7bc35020a1cf571 Mon Sep 17 00:00:00 2001
From: Laurent Bigonville <bigon@bigon.be>
Date: Wed, 2 Nov 2016 14:04:37 +0100
Subject: [PATCH] policycoreutils: Use "new" sepolicy icon in .desktop file for
 sepolicy gui

New hires icons have been added to the repository end 2013 but the
sepolicy .desktop file is not using it, let's fix that.
---
 policycoreutils/sepolicy/sepolicy.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/policycoreutils/sepolicy/sepolicy.desktop b/policycoreutils/sepolicy/sepolicy.desktop
index f9d0cec..181a1c5 100644
--- a/sepolicy/sepolicy.desktop
+++ b/sepolicy/sepolicy.desktop
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=SELinux Policy Management Tool
 Comment=Generate SELinux policy modules
-Icon=system-config-selinux
+Icon=sepolicy
 Exec=/usr/bin/sepolicy gui
 Type=Application
 Terminal=false
-- 
2.10.2

